packages/core/helper-plugin/src/components/DynamicTable/index.js

```diff
@@ -175,7 +188,7 @@ const Table = ({
         ) : (
           Children.toArray(children).map((child) =>
             cloneElement(child, {
-              entriesToDelete,
+              entriesToDelete: selectedEntries,
               onClickDelete: handleClickDelete,
               onSelectRow: handleSelectRow,
```

Comment:
In fact all of this as well is quite delicate. 

It's gonna be awful to type with TS as well because you'll _have_ to declare that the props might be undefined because you'll render the component with out them, then you're just gonna have _thousands_ of `?`. IMO it's a good time to do some cleaning up here.